Exercise9.sh

```diff
@@ -0,0 +1,182 @@
+##------------##
+## Question 1 ##
+##------------##
+
+# Load Data
+data1 = read.csv(file = "ponzr1.csv", header = TRUE)
+
+# Create null function with parameters
+nullnllike <- function(p,x,y){
+  B0 = p[1]
+  sig = exp(p[2])
+  
+  expected = B0
+  
+  nullnll = -sum(dnorm(x=y, mean = expected, sd = sig, log = TRUE))
+  return(nullnll)
+}
+
+# Create mutation function with parameters
+mutnllike <- function(p,x,y){
+  B0 = p[1]
+  B1 = p[2]
+  sig = exp(p[3])
+  
+  expected = B0 + B1*x
+  
+  mutnll = -sum(dnorm(x=y, mean = expected, sd = sig, log = TRUE))
+  return(mutnll)
+}
+
+# Determine Null Hypothesis
+initialnull = c(1,1,1,1,1,1,1,1,1,1)
```

Comment:
This could be prone to error. Try doing something like rep(0,length(data[data$mutation=='WT',]))